README.asciidoc

```diff
@@ -1,16 +1,41 @@
-RPMGrill User Guide
-===================
+RPMGrill
+========
 :doctype: book
 :toc:
 :lang: en
 
+Overview
+--------
+
```

Comment:
How about adding the travis link: https://travis-ci.org/default-to-open/rpmgrill.svg?branch=master